app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
- This whole line can be replaced with just .get() method, since get() returns None by default if there is no such key.
- You would iterate over dictionary keys by default, so there is no need to clearly indicate .keys()
- You should use `is None` instead of `== None` as a general rule